bark/benchmark/benchmark_runner.py

```diff
@@ -18,6 +18,11 @@
 from bark.runtime.scenario.scenario import Scenario
 from bark.core.world.evaluation import *
 
+# ltl lib is only build conditionally
+try:
+  from bark.core.world.evaluation.ltl import *
+except:
+  pass
```

Comment:
not so nice, to handle conditional deps via try and catch, maybe add another evaluation specification not based on string but on type, so instead of "EvaluatorLTL" one can also use EvaluatorLTL

Reply:
Could add an wrapper class for the evaluator that pre-defines which LTL Label are used in this one. Otherwise, do the evaluators have to be added here and can’t we pass them via lambda function? Have to agree that a try and except might not be the best solution. But would be acceptable for me.